image/base/Dockerfile

```diff
@@ -2,6 +2,21 @@
 # VERSION:  release
 
 ARG DEBIAN_RELEASE=bookworm
+FROM discourse/ruby:3.3.6-${DEBIAN_RELEASE}-slim AS builder
```

Comment:
> discourse/ruby:3.3.6-${DEBIAN_RELEASE}-slim

I don't think we need to depend on Ruby here and am wondering if we can just rely on the debian images?

Reply:
It could, it doesn't really matter too much - I figured we're throwing it away after moving the compiled binaries over and building from the ruby base image here is convenient as we already need that image to build all the discourse base image to begin with.